sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE `creature_template` SET `ScriptName`='npc_pet_gen_soul_trader' WHERE `entry`=27914;
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_ethereal_pet_aura','spell_ethereal_pet_onsummon','spell_ethereal_pet_aura_remove','spell_steal_essence_visual');
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES 
+(50051, 'spell_ethereal_pet_aura'),
```

Comment:
remove spaces
should be: (50051,'spell_ethereal_pet_aura')
same below.
Btw, PR is ready right?

Reply:
Done :+1: 
I guess so (works for me).